Sloth.Web/Controllers/JobsController.cs

```diff
@@ -267,7 +272,12 @@ public async Task<IActionResult> RunCybersourceBankReconcile(DateTime date)
                 {
                     // call methods
                     log.Information("Starting Job");
-                    await cybersourceBankReconcileJob.ProcessReconcile(date, record, log);
+                    await foreach (var jobBlob in cybersourceBankReconcileJob.ProcessReconcile(date, record, log))
+                    {
+                        if (jobBlob == null) continue;
+                        // save uploaded blob metadata
+                        dbContext.CybersourceBankReconcileJobBlobs.Add(jobBlob);
+                    }
```

Comment:
Probably not worth changing but fyi my preference here would be to avoid continue; in most instances and instead do 
```
if (jobBlob != null) {
      // save uploaded blob metadata
     dbContext.CybersourceBankReconcileJobBlobs.Add(jobBlob);
}
```

Reply:
In this particular case, it didn't really improve readability. But for multiple levels of if/else branches, I've found inverting the if and returning/breaking/continuing makes a big difference.

Reply:
return/break I totally agree with -- I find continue less easy to reason about and rarely useful.  might just be a personal preference though.